clang/lib/CodeGen/BackendUtil.cpp

```diff
@@ -43,6 +43,7 @@
 #include "llvm/Passes/PassBuilder.h"
 #include "llvm/Passes/PassPlugin.h"
 #include "llvm/Passes/StandardInstrumentations.h"
+#include "llvm/SYCLLowerIR/CleanupSYCLCompilerInternalMetadata.h"
```

Comment:
Nit. For sake of simplicity, this can be CleanupSYCLMetadata?

Reply:
May be, but there are probably other metadata that we would like not to clear

Reply:
I think that is ok. We aren't clearing all internal metadata as well right?

Reply:
Applied, thanks!